src/main/resources/settings.json

```diff
@@ -3,21 +3,23 @@
   "isRemote": false,
   "remoteConnectionUrl": "http://127.0.0.1:4723/wd/hub",
   "screensLocation": "<NAME_OF_PACKAGE_WITH_SCREENS>",
+  "deviceKey": "<KEY_OF_DEVICE_FROM_DEVICES_JSON>",
 
   "driverSettings": {
     "android": {
       "applicationPath": "./src/test/resources/apps/ApiDemos-debug.apk",
       "capabilities": {
+        "platformName": "Android",
+        "automationName": "UiAutomator2",
         "deviceName": "Android Emulator"
```

Comment:
should not we remove this key here because of devices settings should carry about device name?
the same for iOS